src/rebuild/srv.c

```diff
@@ -1134,6 +1134,12 @@ rebuild_task_ult(void *arg)
 		return;
 	}
 
+	rc = rebuild_notify_ras_start(&task->dst_pool_uuid,
+				      RB_OP_STR(task->dst_rebuild_op));
+	if (rc)
+		D_ERROR(DF_UUID": failed to send RAS event\n",
+			DP_UUID(task->dst_pool_uuid));
```

Comment:
if ras_start failed, maybe ras_end() should be skipped?

Reply:
Hmm. It's an INFO-level event and I think it shouldn't be fatal to the operation if it fails for some reason. I agree that it might be a little confusing to see an end event with no matching start, but the end event has the most important information.